test/integration/sonobuoy_integration_test.go

```diff
@@ -32,6 +34,42 @@ func findSonobuoyCLI() (string, error) {
 	return sonobuoyPath, nil
 }
 
+func cleanup(t *testing.T, namespace string) {
+	var stdout, stderr bytes.Buffer
+	deleteCommand := exec.Command(sonobuoy, "delete", "--wait", "-n", namespace)
```

Comment:
If we have various namespaces for different tests (so we wont have collisions) then I'd say we can not do `--wait`, what do you think?

It would speed tests up and avoid issues where a namespace fails to terminate promptly (which isn't uncommon for k8s)

Reply:
You're right, it doesn't need to wait.